Gemfile

```diff
@@ -7,26 +7,23 @@ else
 end
 
 gem 'metadata-json-lint'
-gem 'puppetlabs_spec_helper', '>= 1.1.1'
-gem 'puppet-lint', '>= 1.0.0'
+gem 'puppetlabs_spec_helper', '>= 1.2.0'
 gem 'facter', '>= 2.2.0'
 gem 'rspec-puppet', '~> 2.0'
+gem 'puppet-lint', '>= 1.0', '< 3.0'
 gem 'puppet-lint-absolute_classname-check'
 gem 'puppet-lint-alias-check'
 gem 'puppet-lint-empty_string-check'
 gem 'puppet-lint-file_ensure-check'
 gem 'puppet-lint-file_source_rights-check'
-gem 'puppet-lint-fileserver-check'
 gem 'puppet-lint-leading_zero-check'
 gem 'puppet-lint-spaceship_operator_without_tag-check'
 gem 'puppet-lint-trailing_comma-check'
 gem 'puppet-lint-undef_in_function-check'
 gem 'puppet-lint-unquoted_string-check'
 gem 'puppet-lint-variable_contains_upcase'
 
-if RUBY_VERSION >= '1.8.7' and RUBY_VERSION < '1.9'
-  # rspec must be v2 for ruby 1.8.7
-  gem 'rspec', '~> 2.0'
-  # rake must be v10 for ruby 1.8.7
-  gem 'rake', '~> 10.0'
-end
+gem 'rspec',     '~> 2.0'   if RUBY_VERSION >= '1.8.7' and RUBY_VERSION < '1.9'
```

Comment:
Ha, no. Upgrading all modules to use ruby 2.3.1 and rubocop caught that on the nscd module.